src/Compiler/Checking/ConstraintSolver.fs

```diff
@@ -695,7 +695,7 @@ and SolveTypStaticReq (csenv: ConstraintSolverEnv) trace req ty =
             let vs = ListMeasureVarOccsWithNonZeroExponents ms
             trackErrors {
                 for tpr, _ in vs do 
-                    return! SolveTypStaticReqTypar csenv trace req tpr
+                    do! SolveTypStaticReqTypar csenv trace req tpr
```

Comment:
I'm pretty sure this doesn't cause an early return on the first item in the loop, so `do!` should be clearer.